templates/indieauth-authorize-form.php

```diff
@@ -0,0 +1,53 @@
+<?php
+$errors = new WP_Error();
+login_header(
+	__( 'Authorize', 'indieauth' ),
+	'',
+	$errors
+);
+$current_user = wp_get_current_user();
+$client_id    = wp_unslash( $_GET['client_id'] ); // WPCS: CSRF OK
+$redirect_uri = isset( $_GET['redirect_to'] ) ? wp_unslash( $_GET['redirect_to'] ) : null;
+$scope        = isset( $_GET['scope'] ) ? wp_unslash( $_GET['scope'] ) : null;
+$state        = isset( $_GET['state'] ) ? wp_unslash( $_GET['state'] ) : null;
+$me           = isset( $_GET['me'] ) ? wp_unslash( $_GET['me'] ) : null;
+$token        = compact( 'client_id', 'redirect_uri', 'scope', 'me' );
+$code         = IndieAuth_Authorization_Endpoint::set_code( $current_user->ID, $token );
+$url          = add_query_arg(
+	array(
+		'code'  => $code,
+		'state' => $state,
+	),
+	$redirect_uri
+);
+?>
+<div class="login-info">
+<?php echo get_avatar( $current_user->ID, '78' ); ?>
+<?php
+	printf(
+		'<p>' . __( 'The app <strong>%1$s</strong> would like to access your site, <strong>%2$s</strong> using the credentials of <strong>%3$s</strong>(%4$s).', 'indieauth' ) . '</p>',
```

Comment:
There should be an extra space between `</strong>(`.

Also it would be nice to skip showing the `(%4$s)` part if that is the same value as `%3`, for example when the user has not set a display name so both strings are the username.